openvino/cc/scripts/ccheader.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python3
```

Comment:
do we have readme.md how to use this script? Or it will be a part of user documentation with how to build custom runtime? Maybe at least add a header with brief info with how to use?

Reply:
I added header with brief info.